modules/calcite/src/test/sql/subquery/scalar/test_correlated_subquery_cte.test_ignore

```diff
@@ -1,8 +1,8 @@
 # name: test/sql/subquery/scalar/test_correlated_subquery_cte.test
 # description: Test correlated subqueries
 # group: [scalar]
-# Ignore https://issues.apache.org/jira/browse/IGNITE-15583
```

Comment:
Not actual?

Reply:
Yes, ticket related to GROUP BY issue, byt there is no GROUP BY tests in this file.